MaxText/checkpointing.py

```diff
@@ -75,6 +77,7 @@ def load_state_if_possible(checkpoint_manager: CheckpointManager,
     mesh: a physical TPU mesh
     state_mesh_annotation: a PyTree of sharding rules, matching
       abstract_unboxed_pre_state.
+    log_checkpoint_restore_time: True, to log checkpoint restore time.
```

Comment:
I don't love making changes that make key functions hrader to read and more complicated.

Can't the wait_until_finished give you this feature?

Reply:
I removed the intrusive logging comments. Thanks for your suggestions offline, to try and use load_state_if_possible. I was able to get it working.